.github/workflows/ci.yml

```diff
@@ -455,10 +455,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Check semver
+      - name: Check `tokio` semver
+        uses: obi1kenobi/cargo-semver-checks-action@v2
+        with:
+          rust-toolchain: ${{ env.rust_stable }}
+          package: tokio
+          release-type: minor
+      - name: Check `tokio` semver
```

Comment:
Why? Also both sub-jobs mention Tokio, even though the latter excludes it.

Reply:
Fixed